src/main/java/com/kuljava/swiatwsi/world/VillageController.java

```diff
@@ -0,0 +1,38 @@
+package com.kuljava.swiatwsi.world;
+
+import com.kuljava.swiatwsi.exceptions.VillagesAmountExceededException;
+import com.kuljava.swiatwsi.services.VillageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/villages")
+public class VillageController {
+
+  private VillageService villageService;
+
+  @Autowired
+  public VillageController(VillageService villageService) {
+    this.villageService = villageService;
+  }
+
+  @GetMapping()
+  public List<Village> getAll() {
```

Comment:
Probably in the future we should replace this with DTO object. But for now it's ok